styles/components/_masthead.scss

```diff
@@ -317,23 +317,85 @@
 // Selector Dropdown
 // ==================================================================
 .sprk-c-Masthead__selector-dropdown {
+  background-color: $sprk-white;
   border-bottom: $sprk-masthead-selector-dropdown-border-bottom-mask-open;
-  border-top: 0;
-  border-right: 0;
-  border-left: 0;
-  border-radius: 0;
+  border-radius: $sprk-border-radius;
   box-shadow: $sprk-masthead-selector-dropdown-box-shadow-mask-open;
+  font-size: $sprk-masthead-selector-dropdown-font-size;
+  position: absolute;
   max-width: 100%;
   margin: 0;
   width: 100%;
   top: 0;
   padding: $sprk-masthead-selector-dropdown-padding;
+  z-index: $sprk-layer-height-m;
 }
 
-.sprk-c-Masthead__selector-dropdown .sprk-c-Dropdown__link {
+.sprk-c-Masthead__selector-dropdown .sprk-c-Masthead__selector-dropdown-link {
   padding: $sprk-space-m $sprk-space-misc-a;
 }
 
+.sprk-c-Masthead__selector-dropdown-header {
+  padding: ($sprk-masthead-selector-dropdown-item-padding / 2)
+    $sprk-masthead-selector-dropdown-item-padding;
+
+  // GET RID? Might always have a heading
+  // When the header has a heading, this adjust its padding because the header takes over as the "first item"
+  + .sprk-c-Masthead__selector-dropdown-links {
+    .sprk-c-Dropdown__item:first-child
+      .sprk-c-Masthead__selector-dropdown-link {
+      padding-top: $sprk-masthead-selector-dropdown-item-padding / 2;
+    }
```

Comment:
I think that this doesn't do anything anymore since we're not using `.sprk-c-Dropdown__item` anywhere inside the Masthead Selector.

Reply:
Edit: This might still apply to HTML actually. Let's chat about this.

Reply:
Talked with Jane about this anyways, it made inconsistent padding, so we decided to just go with an even 16px on vertical padding.